src/main/java/by/onliner/util/MoveTo.java

```diff
@@ -0,0 +1,21 @@
+package by.onliner.util;
+
+import by.onliner.driver.DriverManager;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+/***
+ * Wrapper class for actions moves by mouse to the defined element
+ */
+public class MoveTo {
```

Comment:
I think it's worth updating this utility name as when we use its functionality there is taftology. Ex. MoveTo. moveToElementAndClick()

Reply:
What about ActionsHelper?